### PR TITLE
Public README Initial

### PR DESCRIPTION
**Pull request by jr1221**
This is the NER README that will appear when you enter the organization from the public level (which is the default).

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+
+| Project Name | Description | Technologies Used |
+| ------------ | ----------- | ---------------------------- |
+| **Web Development** |
+| [FinishLine](https://github.com/Northeastern-Electric-Racing/FinishLine)   | A full-stack web application which enables organized project management of the team's build cycle, including timeline, task, personnel, BOM, finance, and meeting management. | Typescript, React, Express |
+| [Launchpad](https://github.com/Northeastern-Electric-Racing-Launchpad) (external org) | A git based educational system for understanding and honing basic skills in the world of software. | Git, Typescript, React, Express
+| **Application Software** | | |
+| [Argos](https://github.com/Northeastern-Electric-Racing/Argos) | A full stack data visualization and storage tool enabling real-time processing of car diagnostics for debugging and racetime information, as well as long term cloud management and querying of car statitistics. | Rust, MQTT, Prisma, Timescale/Postgres, Angular, Protobuf
+| [Nero](https://github.com/Northeastern-Electric-Racing/Nero-2.0) | An extensive and intuitive car dashboard. | QT, MQTT
```
Comment (Peyton-McKee) [inline profile/README.md]: C++, QML

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+## Our Active Projects
+
+
+| Project Name | Description | Technologies Used |
+| ------------ | ----------- | ---------------------------- |
+| **Web Development** |
+| [FinishLine](https://github.com/Northeastern-Electric-Racing/FinishLine)   | A full-stack web application which enables organized project management of the team's build cycle, including timeline, task, personnel, BOM, finance, and meeting management. | Typescript, React, Express |
+| [Launchpad](https://github.com/Northeastern-Electric-Racing-Launchpad) (external org) | A git based educational system for understanding and honing basic skills in the world of software. | Git, Typescript, React, Express
+| **Application Software** | | |
+| [Argos](https://github.com/Northeastern-Electric-Racing/Argos) | A full stack data visualization and storage tool enabling real-time processing of car diagnostics for debugging and racetime information, as well as long term cloud management and querying of car statitistics. | Rust, MQTT, Prisma, Timescale/Postgres, Angular, Protobuf
```
Comment (Peyton-McKee) [inline profile/README.md]: don't need /postgres, timescale is built off of Postgres

Comment (Peyton-McKee) [reply profile/README.md]: Typescript

File: profile/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+Our software team is a diverse group of software, computer, and electrical engineers committed to developing quality products and applications. The scope of the software team ranges from writing the code for real-time firmware control of critical automotive systems to developing a fully featured project management dashboard to maintain the timeline and organization required for building a racecar annually.
+
+## Our Active Projects
+
+
+| Project Name | Description | Technologies Used |
+| ------------ | ----------- | ---------------------------- |
+| **Web Development** |
+| [FinishLine](https://github.com/Northeastern-Electric-Racing/FinishLine)   | A full-stack web application which enables organized project management of the team's build cycle, including timeline, task, personnel, BOM, finance, and meeting management. | Typescript, React, Express |
```
Comment (Peyton-McKee) [inline profile/README.md]: Node.js

File: profile/README.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+We are a student-run organization at Northeastern University building all-electric formula-style race cars from scratch to compete in Formula Hybrid + Electric and Formula SAE.
+
+Our software team is a diverse group of software, computer, and electrical engineers committed to developing quality products and applications. The scope of the software team ranges from writing the code for real-time firmware control of critical automotive systems to developing a fully featured project management dashboard to maintain the timeline and organization required for building a racecar annually.
+
+## Our Active Projects
+
+
+| Project Name | Description | Technologies Used |
+| ------------ | ----------- | ---------------------------- |
+| **Web Development** |
```
Comment (Peyton-McKee) [inline profile/README.md]: Lets move this to the center Column

Comment (Peyton-McKee) [reply profile/README.md]: you can also center the text within the column

File: profile/README.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+| [FinishLine](https://github.com/Northeastern-Electric-Racing/FinishLine)   | A full-stack web application which enables organized project management of the team's build cycle, including timeline, task, personnel, BOM, finance, and meeting management. | Typescript, React, Express |
+| [Launchpad](https://github.com/Northeastern-Electric-Racing-Launchpad) (external org) | A git based educational system for understanding and honing basic skills in the world of software. | Git, Typescript, React, Express
+| **Application Software** | | |
+| [Argos](https://github.com/Northeastern-Electric-Racing/Argos) | A full stack data visualization and storage tool enabling real-time processing of car diagnostics for debugging and racetime information, as well as long term cloud management and querying of car statitistics. | Rust, MQTT, Prisma, Timescale/Postgres, Angular, Protobuf
+| [Nero](https://github.com/Northeastern-Electric-Racing/Nero-2.0) | An extensive and intuitive car dashboard. | QT, MQTT
+| **Firmware** | | | 
+| [Shepherd](https://github.com/Northeastern-Electric-Racing/ShepherdBMS-2)  | A from-scratch battery management system.  | C, FreeRTOS |
```
Comment (Peyton-McKee) [inline profile/README.md]: shepherd is not FreeRTOS

File: profile/README.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+| [Cerberus](https://github.com/Northeastern-Electric-Racing/Cerberus)     | A real-time state manager for the car, controlling torque calculations and peripheral functions. | C, FreeRTOS |
+| [Proteus](https://github.com/Northeastern-Electric-Racing/ProteusMC) | A real-time from-scratch dual processor motor controller (pre 1.0). | C, FreeRTOS
```
Comment (Peyton-McKee) [inline profile/README.md]: Missing Closing |

File: profile/README.md (hunk 1 above)
Comment (Peyton-McKee) [reply profile/README.md]: Missing Closing |

File: profile/README.md (hunk 2 above)
Comment (Peyton-McKee) [reply profile/README.md]: Missing Closing |

File: profile/README.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+## Our Active Projects
+
+
+| Project Name | Description | Technologies Used |
+| ------------ | ----------- | ---------------------------- |
+| **Web Development** |
+| [FinishLine](https://github.com/Northeastern-Electric-Racing/FinishLine)   | A full-stack web application which enables organized project management of the team's build cycle, including timeline, task, personnel, BOM, finance, and meeting management. | Typescript, React, Express |
+| [Launchpad](https://github.com/Northeastern-Electric-Racing-Launchpad) (external org) | A git based educational system for understanding and honing basic skills in the world of software. | Git, Typescript, React, Express
+| **Application Software** | | |
```
Comment (Peyton-McKee) [inline profile/README.md]: stay consistent with using the | in the web dev one you don't have the extra |

File: profile/README.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+| [Nero](https://github.com/Northeastern-Electric-Racing/Nero-2.0) | An extensive and intuitive car dashboard. | QT, MQTT
+| **Firmware** | | | 
+| [Shepherd](https://github.com/Northeastern-Electric-Racing/ShepherdBMS-2)  | A from-scratch battery management system.  | C, FreeRTOS |
+| [Cerberus](https://github.com/Northeastern-Electric-Racing/Cerberus)     | A real-time state manager for the car, controlling torque calculations and peripheral functions. | C, FreeRTOS |
+| [Proteus](https://github.com/Northeastern-Electric-Racing/ProteusMC) | A real-time from-scratch dual processor motor controller (pre 1.0). | C, FreeRTOS
+| [Mechanical Sensor Board FW ](https://github.com/Northeastern-Electric-Racing/MSB-FW) | A minimal and dynamic sensor interpreter for CAN bus relaying (pre 1.0). | C, FreeRTOS
+| **Embedded Software** | | |
+| [Odysseus](https://github.com/Northeastern-Electric-Racing/Odysseus)  | A custom Linux operating system supporting our Wireless and Siren initiatives, also hosting Calypso, NERO, and Argos. | HaLow 802.11ah, MQTT, Buildroot, Python
```
Comment (Peyton-McKee) [inline profile/README.md]: missing closing |

File: profile/README.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+| [Calypso](https://github.com/Northeastern-Electric-Racing/Calypso) | A minimal application to process and translate CAN bus packets to MQTT messages using a human readable code-generating configuration. | Rust, SocketCAN, MQTT, Python, Protobuf
```
Comment (Peyton-McKee) [inline profile/README.md]: and vice versa Mqtt to can messages

File: profile/README.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
+<p align="center">
+<img src="https://electricracing.northeastern.edu/wp-content/uploads/2023/02/cropped-NER-logo-white.png" alt="drawing" width="200"/>
+</p>
+
+<p align="center">
+<a href="https://electricracing.northeastern.edu">Website </a> | <a href="https://finishlinebyner.com">FinishLine </a> | <a href="https://www.linkedin.com/company/northeasternelectricracing"> <img src="https://raw.githubusercontent.com/tandpfun/skill-icons/main/icons/LinkedIn.svg" width="16" height="16"> </a>
```
Comment (Peyton-McKee) [inline profile/README.md]: should probably add a picture of the car too

File: profile/README.md [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+| <a href="https://www.instagram.com/nuelectricracing/"> <img src="https://raw.githubusercontent.com/tandpfun/skill-icons/main/icons/Instagram.svg" width="16" height="16"> </a> | <a href="https://nerdocs.atlassian.net/wiki/spaces/NER/overview?homepageId=65735"> <img src="https://cdn.worldvectorlogo.com/logos/confluence-1.svg" width="16" height="16"> </a>
```
Comment (Peyton-McKee) [inline profile/README.md]: it looks like there's an extra space in these two that are being underlined, you can do tags with no content without a closing tag by just doing something like <a href="bla" src="bla" />

File: profile/README.md [hunk 12]
```diff
@@ -0,0 +1,43 @@
+<p align="center">
```
Comment (Peyton-McKee) [inline profile/README.md]: use a div not a p

File: profile/README.md (hunk 10 above)
Comment (Peyton-McKee) [inline profile/README.md]: We do have a FinishLine logo we can use, and then Id wrap the NER logo in a link and have it link to our general website

**Review by Peyton-McKee — CHANGES_REQUESTED** [review]
Looks good

File: profile/README.md (hunk 5 above)
Comment (jr1221) [reply profile/README.md]: fixed. hopefully will be soon tho tons of rewriting has to get started lol.

**Pull request merged**